cmd/create.go

```diff
@@ -136,7 +136,7 @@ func create(ctx *cli.Context) error {
 		return cli.ShowCommandHelp(ctx, "create")
 	}
 
-	cls, err := cluster.NewCluster(driverName, addr, name, configGetter, persistStore)
+	cls, err := cluster.NewCluster(driverName, name, addr, configGetter, persistStore)
```

Comment:
This is kind of a serious bug for kontainer-engine. A separate commit for this change would be more nice.